src/content/docs/2/guide/frontend/trunk.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Trunk
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Trunk is a WASM web application bundler for Rust. Learn more about Trunk at https://trunkrs.dev. This guide is accurate as of Trunk 0.17.5.
+
+## Checklist
+
+- Use SSG, Tauri doesn't officially support server based solutions.
+- Use `address = "0.0.0.0"` so that the webserver is available on the network for mobile development.
+- Use the fork of Trunk made by one of the Tauri core member which contains a couple of fixes to it. Hopefully we can get these merged upstream eventually.
+
+## Example Configuration
+
+<Tabs>
+<TabItem label="Cargo">
+
+```sh
+cargo install --git https://github.com/amrbashir/trunk
+```
+
+</TabItem>
+<TabItem label="Binstall">
+
+```sh
+cargo binstall --git https://github.com/amrbashir/trunk
```

Comment:
Is it common in other Rust projects to document `bininstall` in their steps? I don't see it mentioned in the Trunk docs. By putting this here could it cause potential issues or questions from a reader who tries this and doesn't know what they're doing?

Reply:
Also impacts https://github.com/tauri-apps/tauri-docs/pull/1405/files#diff-20c8897908cc041a1fc1be23670a08a21c4c779cd020a96e6db7622919a83bc8R13

Reply:
`cargo binstall` is probably not supported for my fork anyways, you should use `cargo install --git`.

Reply:
Yea binstall I don't think works either since it relies on finding a prebuilt version in Github releases. I primarily added it as a note to myself to remember binstall for other projects that support it

Reply:
Cool, just updated in the callout and also removed that section. Thanks!